drivers/adc/adc_npcx.c

```diff
@@ -304,12 +298,14 @@ static const struct adc_driver_api adc_npcx_driver_api = {
 
 static int adc_npcx_init(const struct device *dev);
 
-static const struct npcx_alt adc_alts[] = NPCX_DT_ALT_ITEMS_LIST(0);
+PINCTRL_DT_INST_DEFINE(0);
+BUILD_ASSERT(DT_NUM_INST_STATUS_OKAY(DT_DRV_COMPAT) == 1,
+	"only one 'nuvoton_npcx_adc' compatible node present");
```

Comment:
The error message implies that the error is that only one ADC node was found.  Please change to (addition in bold)
"only one 'nuvoton_npcx_adc' compatible node **may be** present"

Please update all the similar BUILD_ASSERTS.

Reply:
Done.